src/analysis/modals/RuntimeErrorModal.ts

```diff
@@ -0,0 +1,77 @@
+import { Modal, ModalProps } from '@terra-ui-packages/components';
+import { useNotificationsFromContext } from '@terra-ui-packages/notifications';
+import _ from 'lodash/fp';
+import { ReactNode, useEffect, useState } from 'react';
+import { div, h } from 'react-hyperscript-helpers';
+import { spinnerOverlay } from 'src/components/common';
+import { LeoRuntimeProvider, RuntimeBasics } from 'src/libs/ajax/leonardo/providers/LeoRuntimeProvider';
+import colors from 'src/libs/colors';
```

Comment:
While we're here, get colors from `useThemeFromContext` instead of a direct import?